client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -56,20 +52,16 @@ function ReaderSubscriptionListItem( {
 				/>
 			</div>
 			<div className="reader-subscription-list-item__byline">
-				<span className="reader-subscription-list-item__site-title">{ <a href={ streamUrl }> { siteTitle } </a> }</span>
+				<span className="reader-subscription-list-item__site-title">
+					{ <a href={ streamUrl } className="reader-subscription-list-item__link"> { siteTitle } </a> }
+				</span>
+				<div className="reader-subscription-list-item__site-excerpt">{ siteExcerpt }</div>
 				{ ! isEmpty( authorName ) &&
-					<span>
-						<span className="reader-subscription-list-item__by-text">
-							{ translate( 'by' ) }
-						</span>
-						<span><a href={ streamUrl }> { authorName } </a></span>
+					<span className="reader-subscription-list-item__by-text">
+						{ translate( 'by' ) }
```

Comment:
this translation should include the anchor component. so `translate( 'by {{author}}', { components: { author: <a href={ streamUrl } className="reader-subscription-list-item__link"> { authorName } </a> } } )`